l10ninsp/slave.py

```diff
@@ -123,6 +132,19 @@ def finished(self, *args):
       rc = FAILURE
     self.sendStatus({'rc': rc})
 
+def compareDirs(path1, path2):
+    cljsPath = '/home/elmo/elmo/slave-ball/vendor-local/compare-locales.js'
+    # nodejs ./bin/compare-dirs.js --data=json ~/elmo/stage/workdir/a
+    process = subprocess.Popen(
+      ["/usr/bin/nodejs",
```

Comment:
We don't have current nodejs packages for lucid, and so I'm using the binary tarball. Newer VMs will likely end up being CentOS, no idea if they follow naming conventions. And if we ever get the automation to run on macs again, it'd also be nice, y'know :-)